lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -432,6 +436,10 @@ public function update_item( $request ) {
 			return new WP_Error( 'rest_comment_invalid_type', __( 'Sorry, you cannot change the comment type.' ), array( 'status' => 404 ) );
 		}
 
+		if ( isset( $request['content'] ) && ! is_string( $request['content'] ) ) {
+			return new WP_Error( 'rest_comment_content_invalid', __( 'Invalid comment content.' ), array( 'status' => 400 ) );
```

Comment:
We are using `wp_filter_post_kses` for the `sanitize_callback` on this, so I'm not sure how this would fail?